src/feature.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable import/prefer-default-export */
+export const checkIfDone = (done) => !done;
+
+const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+export const addTask = (description) => {
+  const index = tasks && tasks.length > 0 ? tasks.length + 1 : 1;
+  const task = {
+    description,
+    index,
+    done: false,
+  };
+  tasks.push(task);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  window.location.reload();
```

Comment:
You could create a function that takes in **tasks** as a parameter for setting the localstorage. You can as well include the window.location.reload() in function.
This function will then set the localstorage and reload the page.